service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementMapper.java

```diff
@@ -213,6 +232,26 @@ private String buildTextDescription(ReferralRequest referralRequest) {
         return text.orElse(StringUtils.EMPTY);
     }
 
+    private List<String> buildPractitionersDescription(ReferralRequest referralRequest) {
+        return referralRequest.hasRecipient()
+            ? referralRequest.getRecipient().stream()
+                .filter(this::isReferenceToPractitioner)
+                .skip(1)
+                .map(this::getRecipientName)
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList())
+            : Collections.emptyList();
+    }
+
+    private String getRecipientName(Reference reference) {
+        final IIdType iidType = reference.getReferenceElement();
+        return messageContext.getInputBundleHolder()
+            .getResource(iidType)
+            .map(Practitioner.class::cast)
+            .map(practitioner -> practitioner.getNameFirstRep().getNameAsSingleString())
```

Comment:
I think this method will map to string also all prefixes and all suffixes. The story says only about name and family name.

Reply:
We're now using `RequestStatementExtractor::extractHumanName`